### ISSUE TITLE
Project & tool naming

### ISSUE THREAD

**Issue by hsablonniere** [ensuite/ensuite-present#3]
This project will hold several tools to help trainers & speakers for:

* Authoring
* Presenting
* Sharing
* Gathering feedbacks

For now, the status of the project only covers presenting on stage or in a classroom. We need a name for the command line tool used to present.

@graphitefriction & @mojavelinux Propose something that sounds french like *montez* but I don't think it works well.

My propositions:

* lutrin ([images](https://www.google.com/search?q=lutrin&tbm=isch))
* pupitre ([images](https://www.google.com/search?q=pupitre&tbm=isch))

They both define a lectern of some kind. The term lectern is already taken on NPM and as a Github org.

**Comment by hsablonniere** [comment]
We also need to consider a different naming for the github project. Is `presentation-tools` too lame and simple?

**Comment by mojavelinux** [comment]
As far as the project name goes, I like "pres-ensuite". It reads like "present(ation) suite", but with a French twist.

**Comment by mojavelinux** [comment]
One idea for the command line tool to present is just "pres". It's short for "present" and also in French "near" (minus the accent). It's also easy to type.

**Comment by mojavelinux** [comment]
I'm not opposed to lutrin. Not a fan of pupitre. However, I think it works better if it's a verb, since the tool represents an action.

**Comment by mojavelinux** [comment]
We could also consider lance or lancez (derived from "to cast").

**Comment by graphitefriction** [comment]
Aww, can we call "it" lutin?!?! I know, totally inappropriate, but I :heart: hobgoblins

![220px-lutin_by_godo](https://cloud.githubusercontent.com/assets/1124042/22674372/e9d672ce-ec9c-11e6-83e1-978fc305633e.jpg)

Anyway, coming back from that tangent... 

What exactly is the "it" we're naming?

Just one of the tools?

> Authoring
> Presenting
> Sharing
> Gathering feedbacks

And if so, which one?

Or the whole bundle of tools?

Or the GitHub project?

Or the command that calls one of the tools?

**Comment by graphitefriction** [comment]
By naming it a name that means lectern, I'm assuming we're just naming the command/tool that the that the trainer invokes at the start of the presentation and uses while he or she is presenting on stage. Yes? No? Am I booted off the island?

**Comment by mojavelinux** [comment]
Yes, that's exactly what we are naming. It's the tool that launches the console application that the browser connects to. There are at least two windows. The speaker view and the projector view. This application supplies the content to both windows.

**Comment by hsablonniere** [comment]
How about having the project called something like `foobar` and all tools called `foobar-create`, `foobar-present`...

**Comment by hsablonniere** [comment]
We could also go for simple keywords like the [hapi](https://github.com/hapijs/hapi) framework and its modules [podium](https://github.com/hapijs/podium), [joi](https://github.com/hapijs/joi), [hoek](https://github.com/hapijs/hoek), [boom](https://github.com/hapijs/boom)...

Something like:

* `crft` (for craft)
* `przt` (for present)
* `shr` (for share)
* `fdbk` (for feedbacks)

All of them are available on NPM. I'm not event sure yet if all of those needs will be CLI tools but we never know.

I like `ensuite` but not `pres-ensuite`. It's a bit too long.

**Comment by mojavelinux** [comment]
I could get behind `ensuite`. It's available!

I like your abbreviated tool names, and I know Sarah will like it too since she abbreviates everything.

I might even bring it down to 3 letters:

* crt (create / craft)
* prz (present)
* fbk (feedback)
* psh (push / publish)

(or they should all be the same number of letters)

But I think the npm names should be namespaced, so it would be ensuite-crt, ensuite-prz, etc.

**Comment by hsablonniere** [comment]
Works for me!

Let me recapitulate:

Project name: rename from `opendevise/presentation-tools-wip` to `opendevise/ensuite`

Present tool, aka *the speaker console* becomes a node module called `ensuite-prz` and can be invoked (once installed globally) as a command line called `prz`?

**Comment by mojavelinux** [comment]
I can confirm we're going with `opendevise/ensuite` as the aggregate project name and I like the cli name as `prz`. I think the module name should be written out as `ensuite-present`. This would be akin to apache-maven as the project name, but `mvn` as the tool name.

While it's fine for now to release the modules out of a mono repository, I do think we should consider migrating to separate repositories for each module. But again, I don't want to add undue complexity to this stage.

**Comment by hsablonniere** [comment]
OK, perfect :+1:

About the mono-repo, I propose that we keep trying to isolate these tools and we'll study the separation in several repos when the problem shows up.

UPDATE: In fact, for now the repo only contains the present tool so we could directly rename it to `opendevise/ensuite-present` right?

**Comment by mojavelinux** [comment]
Correct, let's name the repo so it matches the tool.

**Comment by hsablonniere** [comment]
There's still a few details to discuss regarding project, repo, tool namings. Here are a few comments discussed in PR #7:

* https://github.com/opendevise/presentation-tools-wip/pull/7#discussion_r100918972
* https://github.com/opendevise/presentation-tools-wip/pull/7#issuecomment-279567563
